app/models/patient/consent_status.rb

```diff
@@ -6,7 +6,7 @@
 #
 #  id               :bigint           not null, primary key
 #  academic_year    :integer          not null
-#  disease_types    :enum             is an Array
+#  disease_types    :enum             default([]), is an Array
```

Comment:
It looks like the migration sets this to `NULL`, but it isn't reflected here in the comment.

Reply:
Good spot. Fixed that.